src/runtime/agas/big_boot_barrier.cpp

```diff
@@ -791,9 +791,10 @@ void big_boot_barrier::wait_hosted(
         , unassigned
         , suggested_prefix);
 
-    std::srand(static_cast<unsigned>(util::high_resolution_clock::now()));
+    std::mt19937 gen(std::random_device{}());
```

Comment:
Wouldn't the result returned from `std::random_device` here be sufficient as a random value for the first parcel-id?

Reply:
Sure, will change it :)